main.tex

```diff
@@ -240,8 +240,9 @@ \subsection{Astropy development model}
 maintainability as the code itself.  Without comments there are $142197$ lines
 of code.} contributed by $232$ unique contributors over $19270$ \texttt{git}
 commits.
-The commit distribution \inlinecomment{Tom R}{too vague, maybe 'the distribution of number of commits as a function of ...'} follows an approximate power-law (with log-slope of
-$\approx -0.47$) \inlinecomment{AG}{round to -0.5?  This is astronomy, after all.}, meaning most of the commits were and are contributed by a
+The distribution of number of commits as a function of number of contributors
```

Comment:
Hmm... I would have said the original wording is correct. The data are not the number of commits as a function of <something>, but rather really just the probability distribution of the number of commits any given person has.

Although now that I think about it, I guess it's the same either way?  That is, I think the statement about "approximate power-law" and the value of the slope are the same whether you phrase it as a distribution or a binned function?

Reply:
The original wording was unclear, it just said 'The commit distribution', but it wasn't clear whether you meant the number of commits, and as a function of what. I guess what I mean is that 'The commit distribution' doesn't give me enough information about what x and y is :) Could you suggest an alternative re-wording?

Reply:
What about "The distribution of number of commits of each committer"?  Do you think that's clearer?  To me "the commit distribution" is a shorthand for exactly that.  That is to say, there is *no* "y" per se - it's a probability distribution over number of commits (although I guess you could consider "y" to be "probability"...).

Just to make sure we're on the same page, @astrofrog - the fact that it's not clear to you means it should be clarified - I'm just having trouble knowing what wording  to use because it *already* seems clear to me.

Reply:
@eteq - Just to clarify, I'm trying to figure out how I'd actually do the fitting myself. Should I make a histogram where I bin contributors by number of commits? E.g. 10 authors made 1 commit, 3 authors made 2 commits, 1 author made 3 commits. Or am I showing number of commits versus commit rank? E.g. the most prolific commiter made 1000 commits, the second one made 100 commits, etc. What I'm trying to get at is that there are various diagnostic plots one could make with authors and committers. There has to be a y axis of some kind if you are fitting a power-law.

Just to be clear, I'm suggesting that 'the commit distribution' has as much information in it as 'the star distribution' or 'the galaxy distribution'. 'The commit distribution' could even mean over time.

"The distribution of number of commits of each committer" still doesn't really make it clear to me.

The wording here is especially important because we aren't showing the graph. If we were, then that would probably be immediately clearer. Could you show me the plot you mean?

Reply:
So you said 'it's a probability distribution over number of commits' which I think starts to address my question - but how did you do the fitting? Did you bin the commits?

Reply:
Anyway, show me the plot so I can understand why I'm not understanding :)